routes/podcast.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+const router = express.Router()
+
+const { podcastUpdateSchema, idSchema, podcastSchema } = require('../schemes/podcast')
+// const { validatePodcastID, validatePodcastUpdate, validatePodcast } = require('../middleware/schemaRequests')
+const { urlParamsSchemeValidator, requestSchemeValidator } = require('../middleware/schemaRequests')
+const { updatePodcast, addNewPodcast, deletePodcast, getPodcast } = require('../controllers/podcast')
+router.get('/podcast/:id', [urlParamsSchemeValidator(idSchema)], getPodcast)
```

Comment:
A space is missing here. after require block and between the rest of the code there should be a space